src/plugins/intel_cpu/src/nodes/rope.h

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+#include <ie_common.h>
+#include <node.h>
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "transformations/cpu_opset/common/op/rope.hpp"
+
+namespace ov {
+namespace intel_cpu {
+namespace node {
+
+class RoPE : public Node {
+public:
+    RoPE(const std::shared_ptr<ngraph::Node>& op, const GraphContext::CPtr context);
+
+    void getSupportedDescriptors() override {}
+    bool created() const override {
+        return getType() == Type::RoPE;
+    }
+    bool needPrepareParams() const override {
+        return false;
+    };
+    void executeDynamicImpl(dnnl::stream strm) override {
+        execute(strm);
+    }
+    void initSupportedPrimitiveDescriptors() override;
+    void execute(dnnl::stream strm) override;
+    static bool isSupportedOperation(const std::shared_ptr<const ngraph::Node>& op, std::string& errorMessage) noexcept;
+
+    const RoPENode::Config& getConfig() const {
```

Comment:
minor: getConfig still seems should be in private section.

Reply:
It's not used anymore, removed